tests/sentry/api/test_issue_search.py

```diff
@@ -182,10 +182,9 @@ def test_invalid(self):
 @region_silo_test
 class ConvertQueryValuesTest(TestCase):
     def test_valid_assign_me_converter(self):
-        filters = [SearchFilter(SearchKey("assigned_to"), "=", SearchValue("me"))]
-        expected = value_converters["assigned_to"](
-            [filters[0].value.raw_value], [self.project], self.user, None
```

Comment:
LGTM

Maybe put in the PR description that referencing the SearchValue through the filter was causing the typing issue and it was fixed by avoiding implicit type conversion.